app/styles/base.css

```diff
@@ -582,6 +582,9 @@ select:active {
 #noVNC_setting_path {
   width: 100px;
 }
+.noVNC_disabled_setting {
```

Comment:
Do we still have that? I thought that was removed...

Wherever is fine. Close to the top probably makes sense as it overrides the default text colour and nothing else.